src/style.css

```diff
@@ -0,0 +1,77 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+* {
```

Comment:
Usually we set font styles to `html` tag, other elements will inherit it

Reply:
https://drive.google.com/file/d/1ugcbMyvKyc-FHUDtZvtQ6lGfUrC10LIa/view?usp=sharing
https://drive.google.com/file/d/1N-RO6yuzCaAMUdnaQ2dhls97bveg4f9A/view?usp=sharing

I don't know how it works but after moving fonts styles from tag * to tag html tests failed (
(I didn't change anything more)

Reply:
Strange, it shouldn't work that way 🤔 Okay, let it be in `*` for this task